src/pages/AdminTemplates/AdminTemplates.tsx

```diff
@@ -0,0 +1,154 @@
+import classNames from "classnames";
+import React, { useState } from "react";
+
+import { DEFAULT_MAP_BACKGROUND } from "settings";
+
+import { Room } from "types/rooms";
+
+import { PartyMapContainer } from "pages/Account/Venue/VenueMapEdition";
+
+import "./AdminTemplates.scss";
+
+interface TemplateType {
```

Comment:
```suggestion
export interface TemplateType {
```

Reply:
Also, we generally don't suffix our types with `Type`. Why are we doing that here?

It looks like this should just be `Template` or similar.